N/NUMA/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "NUMA"
```

Comment:
Should this be called `numactl`?

Reply:
It is more about `libnuma` than it is about `numactl`